plan/explain_test.go

```diff
@@ -11,18 +11,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package executor_test
+package plan_test
 
 import (
 	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/util/testkit"
 	"github.com/pingcap/tidb/util/testleak"
 )
 
-func (s *testSuite) TestExplain(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
+var _ = Suite(&testExplainSuite{})
+
+type testExplainSuite struct {
+}
+
+func (s *testExplainSuite) TestExplain(c *C) {
+	store, err := newStoreWithBootstrap()
+	c.Assert(err, IsNil)
+	tk := testkit.NewTestKit(c, store)
 	defer func() {
```

Comment:
I guess you should `defer store.Close()`, and `defer domain.Close()`, otherwise testleak won't happy.